bioio/plugins.py

```diff
@@ -37,7 +38,9 @@ class PluginEntry(NamedTuple):
 
 
 # global cache of plugins
-plugins_by_ext_cache: Dict[str, List[PluginEntry]] = {}
+plugins_by_ext_cache: Union[
+    Dict[str, List[PluginEntry]], OrderedDict[str, List[PluginEntry]]
```

Comment:
should only be OrderedDict, right?

Reply:
I believe you are correct but might need to make the function have a plugins_by_ext_unordered variable in addition to appease mypy. Will give it a whirl.